ParcialProgramacion4/src/Empresa.java

```diff
@@ -21,6 +21,66 @@ public Empresa(Scanner scanner) {
         this.convocatorias = new ArrayList<Convocatoria>();
         this.habilidades = new ArrayList<Habilidad>();
     }
+    
+    //AGREGADO --- DAR DE BAJA EMPLEADO
+    public void darDeBajaEmpleado() {
+    	System.out.println("\nNumero de legajo empleado a eliminar: ");
+    	int legajo = Integer.parseInt(scanner.nextLine());
+    	
+    	Empleado empleadoEliminar = this.buscarEmpleado(legajo);
+    	
+    	if (empleadoEliminar != null) {
+    		System.out.println("\n----------------------------------------");
+        	System.out.println("Información empleado a eliminar: ");
+        	empleadoEliminar.mostrar();
+        	System.out.println("----------------------------------------\n");
```

Comment:
Yo haria que los mostrar tengan los divider, y que le puedas mandar un titulo personalizado, o sino que te tire uno default
```suggestion
    		Logger.divider();
        	System.out.println("Información empleado a eliminar: ");
        	empleadoEliminar.mostrar();
        	Logger.divider();
```